### DISCUSSION TITLE
Mobile views for PrimeBlocks and Figma UI Kit

### DISCUSSION THREAD

**Discussion by dbsmalltweaks** [primefaces/community#1483] [Figma UI Kit] [answered]
Our designers need to design our mobile experience and website experiences separately, and we're relying heavily on PrimeBlocks to get the initial release done as quickly as possible. There are definitely pre-determined mobile behaviors for PrimeBlocks in the browser, but there are no mobile mockups of the PrimeBlocks in the Figma libraries we purchased.

The only way our designers can determine the default behavior is to go to the browser and manually resize their screens. They then have to go into Figma and manually recreate the mobile views based on observed behavior and hope they got it right. It seems backwards given that we purchased PrimeBlocks, which includes an extensive Figma library. 

Are we missing something?

**Comment by onursenture** [comment, marked as answer]
We created PrimeBlocks' Figma designs not for the customers who purchase them but for our teams implementing the blocks. However, we also decided to share this internal file with users in response to requests. Therefore, the file has evolved according to our own needs. 

So far, we have yet to receive many requests for mobile versions, and it will not be easy to add mobile designs for the nearly 500 blocks from scratch. Therefore, we are not currently considering allocating resources for this. 

Thank you for your interest and feedback.

**Comment by kleinpetr** [comment]
Hi any news on mobile views of PrimeBlocks?

**Reply by onursenture** [reply]
Hi, There is no change on this matter; we do not have a plan to add mobile views to PrimeBlocks Figma.
